### PR TITLE
More logging.

### PR DESCRIPTION
**Pull request by jortel**
Add base reconciler to deduplicate fields and begin/end reconcile behavior and refit controllers.
Add additional logging statements to `controller.go` in each controller.
Add additional logging statements to other top level packages.

Requires:
- https://github.com/konveyor/controller/pull/65
- https://github.com/konveyor/controller/pull/66

### CODE REVIEW

File: pkg/controller/migration/controller.go [hunk 1]
```diff
@@ -103,55 +97,42 @@ var _ reconcile.Reconciler = &Reconciler{}
...
-	fastReQ := reconcile.Result{RequeueAfter: FastReQ}
-	slowReQ := reconcile.Result{RequeueAfter: SlowReQ}
-	noReQ := reconcile.Result{}
-	result = noReQ
-
-	r.log = logging.WithName(
+	r.Log = logging.WithName(
 		names.SimpleNameGenerator.GenerateName(Name+"|"),
-		"migration",
+		"map",
```
Comment (mansam) [inline pkg/controller/migration/controller.go:110]: This looks like a possible cut and paste error

File: pkg/controller/provider/controller.go [hunk 2]
```diff
@@ -131,72 +126,57 @@ var _ reconcile.Reconciler = &Reconciler{}
...
-	fastReQ := reconcile.Result{RequeueAfter: FastReQ}
-	slowReQ := reconcile.Result{RequeueAfter: SlowReQ}
-	noReQ := reconcile.Result{}
-	result = noReQ
-
-	r.log = logging.WithName(
+	r.Log = logging.WithName(
 		names.SimpleNameGenerator.GenerateName(Name+"|"),
-		"provider",
+		"hook",
```
Comment (mansam) [inline pkg/controller/provider/controller.go:142]: This looks like a probable cut and paste error, hook -> provider

**Review by mansam — CHANGES_REQUESTED** [review]
Looks great other than a couple possible copy/paste errors.

File: pkg/controller/migration/controller.go (hunk 1 above)
Comment (jortel) [reply pkg/controller/migration/controller.go]: Fixed.

File: pkg/controller/provider/controller.go (hunk 2 above)
Comment (jortel) [reply pkg/controller/provider/controller.go]: Fixed.

**Review by mansam — APPROVED** [review]

**Pull request merged**